lib/daru/vector.rb

```diff
@@ -59,6 +59,8 @@ def map!(&block)
     # Store a hash of labels for values. Supplementary only. Recommend using index
     # for proper usage.
     attr_accessor :labels
+    # Store vector data in an array
+    attr_accessor :data
```

Comment:
Sorry for seeing this so late but this should be `attr_reader`. You don't want to allow users to manipulate the `@data` variable directly.